library/templates/dcp_mappluto_wi.yml

```diff
@@ -4,7 +4,7 @@ dataset:
   acl: public-read
   source:
     url:
-      path: https://www1.nyc.gov/assets/planning/download/zip/data-maps/open-data/nyc_mappluto_{{ version }}_unclipped_shp.zip
+      path: https://s-media.nyc.gov/agencies/dcp/assets/files/zip/data-tools/bytes/nyc_mappluto_{{version}}_unclipped_shp.zip
```

Comment:
Add spaces between {{ version }}

Reply:
i think functionally is the same. But yeah I agree it is more readable with spaces so I am adding it